homeassistant/components/octoprint/binary_sensor.py

```diff
@@ -22,14 +22,15 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     devices = []
     for octo_type in monitored_conditions:
```

Comment:
Iterate over the entity description and see if the key of those are in the monitored_conditions instead.